.github/workflows/maven.yml

```diff
@@ -21,34 +21,45 @@ on: [pull_request]
 
 jobs:
   linux:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
I would recommend not to use "latest". This could cause and non determistic behavior of the pipeline. Maybe some future version of ubuntu breaks the functionality here. So you will need to fix this before you can use the pipeline. 
For this I would prefer to use a fixed version which will cause always the same run results.